docker-contract/docker-compose.yml

```diff
@@ -0,0 +1,73 @@
+version: "3.7"
+
+networks:
+  mojaloop-pisp-net:
+    name: mojaloop-pisp-net
+
+services:
+  mojaloop-testing-toolkit:
+    image: mojaloop/ml-testing-toolkit:v10.3.3-snapshot
+    container_name: mojaloop-testing-toolkit
+    env_file: ./ml-testing-toolkit/local.env    
+    volumes:
+      - "./ml-testing-toolkit/spec_files:/opt/mojaloop-testing-toolkit/spec_files"
+      - "./ml-testing-toolkit/secrets:/opt/mojaloop-testing-toolkit/secrets"
+    ports:
+      - "15000:5000"
+      - "5050:5050"
+    command:
```

Comment:
can you try: `command: npm start` instead of the multiline sh'ed version?

Reply:
Ok